conf/citest/pipelines/conftest.py

```diff
@@ -0,0 +1,17 @@
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Local hook, fixture and plugin requirements for the CITest suite."""
```

Comment:
In the end there is only a plugin, right ? Maybe simplify the doc ?

Reply:
I left it like this because that's the general purpose of this file. But happy to leave it as a description of what it actually has now.